MPM/mpm.php

```diff
@@ -96,8 +96,10 @@ function mpm_webhook()
 		$order_id	= $payment->metadata->order_id;
 		$order		= new WC_Order( $order_id );
 
-		// Don't update an already paid order
-		if (!$order->needs_payment())
+		$onHold = $order->get_status() == 'on-hold';
+
+		// Don't update an already paid order, needs_payment() searches by valid_statusses, but does not include on-hold status, so we add it here.
+		if (!$order->needs_payment() && !$onHold)
```

Comment:
`$order->get_status()` is niet in elke WooCommerce versie beschikbaar. In mijn versie (2.1.10) is deze methode niet beschikbaar.